docs/source/_templates/objective_class.rst

```diff
@@ -0,0 +1,37 @@
+{{ fullname | escape | underline}}
```

Comment:
Previously `class.rst` was the default template for `Objective` subclasses and it only included methods and were missing class attributes. This is a new rst for the subclass.